scene/gui/video_stream_player.cpp

```diff
@@ -189,6 +190,12 @@ void VideoStreamPlayer::_notification(int p_notification) {
 			}
 		} break;
 
+		case NOTIFICATION_UNSUSPENDED: {
+			if (!get_tree()->is_paused()) {
+				_notification(NOTIFICATION_UNPAUSED);
```

Comment:
You could make a fallthrough instead.

Reply:
How would that work with the pause condition?

Reply:
```C++
if (get_tree()->is_paused()) {
	break;
}
[[falthrough]];
```
The `NOTIFICATION_UNPAUSED` is right below.